src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/ProjectDebugPropertyPage.xaml

```diff
@@ -31,6 +31,7 @@
       <ValueEditor EditorType="MultiLineString">
         <ValueEditor.Metadata>
           <NameValuePair Name="UseMonospaceFont" Value="True" />
+          <NameValuePair Name="TextWrapping" Value="Wrap" />
```

Comment:
I think we also need this in the executable launch profile XAML.

Reply:
Yup, you're right.